python/FactorySystem/Parser.py

```diff
@@ -132,6 +140,12 @@ def _parseNode(self, filename, node):
       # Put it in the warehouse
       self.warehouse.addObject(moose_object)
 
+    # Are we in a tree node that "looks" like it should contain a buildable object?
+    elif len(node.params.keys()):
+      print 'Error detected when parsing file "' + os.path.join(os.getcwd(), filename) + '"'
+      print '       Missing "type" parameter in block'
+      error_code = error_code | 0x20
```

Comment:
Looks a bit... archaic to play with bitwise operators. Python has a _set_ datatype, just saying...